docker-compose.yml

```diff
@@ -121,3 +121,33 @@ services:
     privileged: true
     image: multiarch/qemu-user-static
     command: --reset -p yes
+  opensuse_tumbleweed:
+    container_name: opensuse_tumbleweed
+    build:
+      context: .
+      dockerfile: ci/Dockerfile-opensuse
+      args:
+        CONTAINER_IMAGE: opensuse/tumbleweed
+    environment:
+      TOXENV: ${TOXENV-py37-cov,py27-cov}
+    command: tox
+  opensuse_leap_15_1:
+    container_name: opensuse_leap_15_1
+    build:
+      context: .
+      dockerfile: ci/Dockerfile-opensuse
+      args:
+        CONTAINER_IMAGE: opensuse/leap:15.1
+    environment:
+      TOXENV: ${TOXENV-py36-cov,py27-cov}
+    command: tox
+  opensuse_leap_15_0:
+    container_name: opensuse_leap_15_0
+    build:
+      context: .
+      dockerfile: ci/Dockerfile-opensuse
+      args:
+        CONTAINER_IMAGE: opensuse/leap:15.0
+    environment:
+      TOXENV: ${TOXENV-py36-cov,py27-cov}
+    command: tox
```

Comment:
Can you move the suse cases to above "qemu" below "ubuntu_trusty", because "qemu" is specific docker-compose task used for multi-arch container cases.